voidcrew/code/modules/factions/_common/faction_datum.dm

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Some dmdocs throughout this whole file would be very much appreciated.
```
/** 
 *
 */
```
for multi line pre proc comments,
```
/// comment
```
for single var comments